src/App.js

```diff
@@ -1,23 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
```

Comment:
- [ ] Please implement routes for your applications so users can switch between pages.